packages/terra-site/src/examples/dynamic-grid/ResponsiveGrid.jsx

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+import DynamicGrid from 'terra-dynamic-grid';
+
+import Card from './Card';
+
+const template = {
+  'grid-template-columns': '1fr 1fr 1fr 1fr 1fr',
+  'grid-template-rows': '100px',
+  'grid-gap': '10px',
+};
+
+const region1 = {
+  defaultPosition: {
```

Comment:
I'm actually happy with how this reads.. I think most scenarios will have a single default template.. with regions having positions across multiple breakpoints configured onto that template.